README.md

```diff
@@ -8,7 +8,6 @@ Light Gradient Boosting Machine
 [![Static Analysis GitHub Actions Build Status](https://github.com/microsoft/LightGBM/workflows/Static%20Analysis/badge.svg?branch=master)](https://github.com/microsoft/LightGBM/actions)
 [![Azure Pipelines Build Status](https://lightgbm-ci.visualstudio.com/lightgbm-ci/_apis/build/status/Microsoft.LightGBM?branchName=master)](https://lightgbm-ci.visualstudio.com/lightgbm-ci/_build/latest?definitionId=1)
 [![Appveyor Build Status](https://ci.appveyor.com/api/projects/status/1ys5ot401m0fep6l/branch/master?svg=true)](https://ci.appveyor.com/project/guolinke/lightgbm/branch/master)
-[![Travis Build Status](https://travis-ci.org/microsoft/LightGBM.svg?branch=master)](https://travis-ci.org/microsoft/LightGBM)
```

Comment:
Also remove

https://github.com/microsoft/LightGBM/blob/792c930305a818db1463911c2cbee92d462eaab9/R-package/.Rbuildignore#L49
https://github.com/microsoft/LightGBM/blob/eb9bbfb327815e843b808f5e2b4d73ee3201b4ac/.ci/test_r_package.sh#L156-L164

Reply:
I'm not convinced we should remove that check in `test_r_package.sh`. Yes the comment references Travis, but the `R-package` jobs have been running for a few months on only GitHub Actions, with no timeout issues. Since they've been stable, I'd rather not touch that check right now.

Reply:
OK, as you wish 🙂 .

Reply:
I'd at least like to test a lot of consecutive builds without it to feel confident that it can be removed. I'd like to do that at some point, but right now I'd rather not delay this PR for it since the change isn't really related to removing Travis.

Reply:
this PR removes travis.yml from Rbuildignore: https://github.com/microsoft/LightGBM/pull/3672/commits/6e86d7d17efa5a8a9e4738495cb561b2414008e0

Reply:
this rule in Rbuildignore was apparently preventing some R CMD CHECK issues:

```text
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  src/external_libs/fmt/.travis.yml
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.

CRAN-pack does not know about
  src/external_libs/fmt/.travis.yml
```

for now I'll just add a specific rule to Rbuildignore for that file

Reply:
done in https://github.com/microsoft/LightGBM/pull/3672/commits/51eddaf872de53611e43f99b522d71a52ba0e46d, and the R builds all passed so I think it's ok